CHANGELOG.md

```diff
@@ -69,7 +85,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [1.0.9] - 2024-05-16
 
-### Update
+### Changed
```

Comment:
nit: I think these and those below would be better as Security tags, but this is fine.